pkgs/top-level/all-packages.nix

```diff
@@ -5295,6 +5295,8 @@ with pkgs;
 
   epubcheck = callPackage ../tools/text/epubcheck { };
 
+  epub-thumbnailer = callPackage ../tools/graphics/epub-thumbnailer { };
```

Comment:
Maybe use `python3Packages.callPackage` and then you can import plover directly rather than through `python3Packages` ?

https://github.com/NixOS/nixpkgs/commit/bd250532981d7a1fa6fbab117b2424c170cd786f

Reply:
hm lemme try, I didn't even know about this option.

Reply:
Fixed!